src/pages/Locations/LocationDetails/components/LocationPhotosGrid/LocationPhotosGrid.jsx

```diff
@@ -0,0 +1,197 @@
+import * as React from "react";
+import {useState} from "react";
+import {AppBar, Button, Dialog, Skeleton, Slide, Toolbar, Typography} from "@mui/material";
+import {ViewGridIcon} from "@heroicons/react/solid";
+import CloseIcon from '@mui/icons-material/Close';
+import './LocationPhotosGrid.css'
+import {Carousel} from "react-responsive-carousel";
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+
+
+const placeholderImages = [
+    'https://via.placeholder.com/1080x900.png?text=Loading...',
+]
+const Transition = React.forwardRef(function Transition(props, ref) {
+    return <Slide direction="up" ref={ref} {...props} />;
+});
+const LocationPhotosGrid = (props) => {
+    const [gallery, setGallery] = useState(false)
+    const [loaded, setLoaded] = useState(false)
+    const handleGalleryOpen = () => {
+        setGallery(true)
+    }
+    const handleGalleryClose = () => {
+        setGallery(false)
```

Comment:
Codacy has a fix for the issue: [Replace `····setGallery(false)` with `setGallery(false);`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
    setGallery(false);
```